### ISSUE TITLE
Recruit participants

### ISSUE THREAD

**Issue by mankoff** [IACS-cryo/IceBB#2]

**Comment by charlesamory** [comment]
charlesamory

**Comment by hgoelzer** [comment]
hgoelzer

**Comment by gunterl** [comment]
gunterl

**Comment by mankoff** [comment]
Other places to recruit participants and announce the project:

+ [ ] CRYOLIST
+ [ ] Polar Impact Slack
+ [ ] Twitter
+ [ ] Mastodon

**Comment by mankoff** [comment]
Everyone who entered a GitHub username now has been invited to join the IACS Organization.

In addition to recruiting from / announcing on the places in the comment above this, we could also send targeted invites to relevant scientists. For example this paper https://doi.org/10.5194/tc-17-4549-2023 ?

Postings should also be broadly phrased: "If you have any interest or recent work dealing with the ice sheet periphery, locations of glacier termini, ice piracy between basins, sector, region, or basin-scale mass balance, etc. please read on..." ?

**Comment by fmaussion** [comment]
yes! This is a good idea. Contacting key IMBIE authors would also probably be useful

**Comment by fmaussion** [comment]
Hiya! They would need to send an email to Ken with their github handle! The table link is available from the drive link: https://github.com/IACS-cryo/Delineation-WG 

(@mankoff , the readme could indicate how to join the group as well)

**Comment by mankoff** [comment]
Actually there is no email needed (I think). 

Initially I added people to this GitHub Organization, but anyone can follow and comment/discuss/etc. all you need is a GitHub account. I've updated the README with more explicit 'how to join this project' instructions.

If we start writing code here, and want to have fine-tuned control over who can commit vs. merge vs. whatever, then maybe people need to join the Organization, but I'm not sure we'll do that level of work here or if we need that level of control.
